### ISSUE TITLE
Guake keeps reopening closed split terminals (grid) on reload of guake

### ISSUE THREAD

**Issue by kolbaba** [Guake/guake#1865]
Guake v3.6.3
OS: Ubuntu 21.04 x86_64 
DE: GNOME 3.38.4 

Reproduction:

I split out 4 terminals:
![before](https://user-images.githubusercontent.com/3298804/125178462-ae4e7780-e1aa-11eb-99a2-999a0e1d5592.png)

I then right click 3/4 of those terminals > "close terminal".

It looks like this:

![after](https://user-images.githubusercontent.com/3298804/125178467-c0301a80-e1aa-11eb-92ef-211e60514b82.png)

Then if I close guake (killall guake) or restart my computer, I see this again:
![before](https://user-images.githubusercontent.com/3298804/125178462-ae4e7780-e1aa-11eb-99a2-999a0e1d5592.png)

Basically I want one terminal window, for special use cases I occasionally like to have multiple splits. For whatever reason it's defaulting to 4 columns/terminals now.

**Comment by kolbaba** [comment]
To solve this I unchecked "Restore previous session (tabs)" under General > Startup & Tabs (Experiment). Closing.
![Screenshot from 2021-07-11 11-24-21](https://user-images.githubusercontent.com/3298804/125202740-93c3df00-e23a-11eb-9071-b37acc1ce0db.png)
